mivs/templates/mivs_base.html

```diff
@@ -0,0 +1,9 @@
+{% extends "preregistration/preregbase.html" %}
```

Comment:
If you're not familiar with Django, we use Django templates, and this is how template inheritance works in Django.